packages/server/prod-server/src/libs/serverTiming.ts

```diff
@@ -0,0 +1,34 @@
+import { ServerTiming as IServerTiming } from '@modern-js/types';
+
+interface Res {
+  getHeader: (name: string) => number | string | string[] | undefined;
+  setHeader: (name: string, value: string) => void;
+}
+
+const SERVER_TIMING = 'Server-Timing';
+
+export class ServerTiming implements IServerTiming {
+  meta: string;
+
+  res: Res;
+
+  constructor(res: Res, meta: string) {
+    this.meta = meta;
+    this.res = res;
+  }
+
+  addServeTiming(name: string, dur: number, desc?: string) {
+    const _name = `bd-${this.meta}-${name}`;
```

Comment:
modern.js should not be any 'bytedance' content. We can mark Todo here and do some optimization later. For example a new Server architecture, then we can easily complete the extend internally

Reply:
we can distinguish it by metaName.`bd` be injected, only when meta !== 'modern.js'. 

But the code of `modern.js` would also exist `bytedance`, otherwise we overrides the serverTiming struct on inner framework.

Reply:
I prefer to overrides the `serverTiming` struct, but we can remark a TODO first.

Reply:
Remark a TODO, done~